include/openthread/dnssd_server.h

```diff
@@ -147,20 +147,32 @@ typedef enum
     OT_DNSSD_QUERY_TYPE_RESOLVE_HOST = 3, ///< Service type resolve hostname.
 } otDnssdQueryType;
 
+/**
+ * Includes the queries/responses/totalfailures handled by Upstrea DNS server.
```

Comment:
```suggestion
 * Includes the count of queries, responses, failures handled by upstream DNS server.
```
I think `/` generally means 'or'.